mycroft/skills/settings.py

```diff
@@ -242,7 +242,7 @@ def _load_uuid(self):
         uuid_file = join(directory, 'uuid')
         uuid = None
         if isfile(uuid_file):
-            with open(uuid_file, 'r') as f:
+            with open(uuid_file, 'r', encoding='utf8') as f:
```

Comment:
I actually hit errors with the uuid file which is what caused this series in the first place

Reply:
That's pretty strange because all characters in the uuid should be decodable with pretty much any codec.

What system are you running mycroft on?

Can you give me the output of `python3 -c "import locale; print(locale.getpreferredencoding())"` so I can see if I can repeat the issue.

Reply:
Sounds like a cool project. And non-standard locale settings makes more sense in this case...

Reply:
# python3 -c "import locale; print(locale.getpreferredencoding())"
ANSI_X3.4-1968

Reply:
I've tested this with ansi encoding and for me there is no issues.